main.go

```diff
@@ -14,32 +12,6 @@ var VERSION string
 
 // The main entrypoint for Terragrunt
 func main() {
-	defer errors.Recover(checkForErrorsAndExit)
-
 	app := cli.CreateTerragruntCli(VERSION, os.Stdout, os.Stderr)
-	err := app.Run(os.Args)
-
-	checkForErrorsAndExit(err)
-}
-
-// If there is an error, display it in the console and exit with a non-zero exit code. Otherwise, exit 0.
-func checkForErrorsAndExit(err error) {
-	if err == nil {
-		os.Exit(0)
-	} else {
-		logger := util.CreateLogger("")
-		if os.Getenv("TERRAGRUNT_DEBUG") != "" {
```

Comment:
The docs may mention this, so you'll have to update them to whatever env var gruntwork-cli uses